Library/Homebrew/dev-cmd/generate-formula-api.rb

```diff
@@ -74,6 +78,7 @@ def generate_formula_api
         raise
       end
 
+      File.write("formula.json", JSON.generate(api_json))
```

Comment:
I'm not sure where this file should go.

Reply:
Location seems fine. Filename could be something like `formula_internal.json` or `formula_slim.json` or something?

Reply:
Maybe they should both go in the same directory? `internal/formula.json` and `internal/cask.json`